.github/CODEOWNERS

```diff
@@ -6,16 +6,11 @@
 # Everything that falls through the cracks:
 * @electron/reviewers
```

Comment:
This is a pet-peeve of mine, can we remove this catch-all, it means that every PR we end up "participating" in which makes it hard to filter notifications.

People should be able to review PRs without being mentioned like this